go/cmd/schemas_compiler/docker/Dockerfile

```diff
@@ -33,4 +33,5 @@ RUN go install google.golang.org/protobuf/cmd/protoc-gen-go@${PROTOC_GEN_GO_VERS
 
 # use --break-system-packages to avoid "pip error "externally managed environment" on latest python/pip
 # ftr, the alternative is using venv, but it's more complicated
-RUN pip install grpcio-tools${GRPCIO_TOOLS_VERSION} --break-system-packages
+# TODO: uncomment --break-system-packages once we upgrade pip to > 23.0.1 https://pip.pypa.io/en/stable/news/#v23-0-1
+RUN pip install grpcio-tools${GRPCIO_TOOLS_VERSION} #--break-system-packages
```

Comment:
why not just upgrading pip? shouldn't it be done using the command in line 29

Reply:
We use pip 22.3 in other images and I didn't want to upgrade pip here and not there as well (half job)
I now see that apt installs pip 20.3.4 so we should upgrade either way. Should we upgrade to 23.x or align to MLRUN_PIP_VERSION variable?